### DISCUSSION TITLE
今天是不是墙升级了，挂了3台vps，都是本地tcp端口丢弃，vps收不到数据包

### DISCUSSION THREAD

**Discussion by 1265578519** [v2fly/v2ray-core#2358] [General]
今天是不是墙升级了，挂了3台vps，都是本地tcp端口丢弃，vps收不到数据包
各位有这种情况吗？

帖子补充，目前已知
针对普通ws的非https流量，仅封锁去程端口（小概率性被检测到，永久性封端口）
针对https加密流量，封ip（基本百分百检测到，不封端口，24小时后有概率性自动解封ip，443恢复可用）
被检测到不是立即封ip而是记录起来隔天早上统一时间段去执行封禁，都是每天早上5点开始到7点执行封禁端口或ip
缓解方法，加上cdn后，墙已经不知道你的服务器真实ip了。正如v2开发者表示 被墙的都是没用cdn隐藏ip保护

本来还想说试试自动 换端口的，，，搜了一下发现别人已经试过了，都是秒封
https://github.com/v2fly/v2ray-core/issues/2363
不知道纯tcp流量会怎么样，不进行ws伪装域名之类，会不会好一点
但是v2的纯tcp流量抓包发现首个数据包有个固定的特征码用于验证，有个固定的头部，目前已知的是SS AEAD不受任何影响

帖子补充
测试了，纯tcp也不行，基本也是秒封，而且是和tls一样，直接封ip地址。

**Comment by laowanwan** [comment]
变智能了。443被BAN，换IP后起流量不到1分钟又被BAN，过几分钟没流量又恢复，再起流量再被BAN，玩不过了。不过如果走V6还可以

**Comment by changkaizhao** [comment]
是的,昨天端口被封，换了端口，今天发现除了443 80 22 其他都被墙了 :(

**Comment by mynameiscaa** [comment]
我也是挂了3台😭 vless+ws+tls

**Comment by mydogshitgold** [comment]
If you are using WebSocket or gRPC transport, try relaying them through major CDNs. They may be able to resurrect your server. 

Also, in case of an emergency, don't forget that you can [ssh into GitHub codespaces](https://docs.github.com/en/codespaces/developing-in-codespaces/using-github-codespaces-with-github-cli) and do [magical things](https://wiki.archlinux.org/title/VPN_over_SSH) with them.

**Reply by cross-hello** [reply]
Oh, just set up a codespace and stand server port foward, it gets closed for connection timeout 😅

**Reply by cross-hello** [reply]
OK, success
![image](https://user-images.githubusercontent.com/42733664/221803447-69e8dd16-797b-4f9a-8a73-a20df9ece191.png)

A tiny toy, indeed don't worth to play

**Reply by mydogshitgold** [reply]
Depends on how you use it.

I can get 160 Mps / 80ms out of it with 4 simultaneous TCP connections.

> It gets closed for connection timeout

Decrease `ServerAliveInterval`.

**Reply by cross-hello** [reply]
This time open SSH mulplex(but the way, how to configure TCP concurrent thread in SSH), and speed up twice.
![image](https://user-images.githubusercontent.com/42733664/221810001-0b39d894-8353-4afc-aab9-1c78cb74bd0a.png)

But just could get about 5% using percent of the capable Internet.

**Reply by mydogshitgold** [reply]
Don't use SSH multiplex, which opens only one TCP connection. Open multiple SSH sessions with simple port forwarding to avoid congestion.

Basicly you want to run `ssh -TNL local_port:host:remote_port multiple times.

**Reply by cross-hello** [reply]
(Why don't you use `ssh -TNL local_port:host:remote_port -L local_port_B:host:remote_port_B -L local_port_C:host:remote_port_C` ?)
All right, after set up multiply SSH client listen to same port, speed remains.(has closed ssh multiplex)
![image](https://user-images.githubusercontent.com/42733664/221815302-5df4fd98-586e-47d9-a7f9-ce1696aa4b32.png)

[cry] it is not a good toy

**Reply by mydogshitgold** [reply]
>(Why don't you use ssh -TNL local_port:host:remote_port -L local_port_B:host:remote_port_B -L local_port_C:host:remote_port_C ?)

Because that command still opens **single** TCP connection. You need to open **multiple** TCP connections to avoid congestion.
And use load balancing on v2ray to increase overall throughput.

**Reply by cross-hello** [reply]
what? Don't you talk about just ssh?
If use manageable server, the speed must will boost up.
![image](https://user-images.githubusercontent.com/42733664/221817577-68fa11e9-e771-448d-8266-2ebe00b51cce.png)

**Reply by cross-hello** [reply]
Ok, don't talk about it.

A boring toy

**Reply by mydogshitgold** [reply]
I am talking about a backup for people who can not access their server.

**Reply by mydogshitgold** [reply]
> what? Don't you talk about just ssh?
> If use manageable server, the speed must will boost up.

Doesn't look like anything to me.

**Comment by daniyyer** [comment]
vless+ws+tls 被封了 vultr的服务器

**Comment by imuxin** [comment]
连续两天早上起来发现端口都不可用了 🐶 。

**Reply by 1265578519** [reply]
对，昨天早上封了一批，今天早上又封了一批。。。离谱啊

**Reply by changkaizhao** [reply]
连续三天换端口，每次减持到午夜，端口再次被打死，测试并没有所有端口封住。443，80，还有一个其他服务的5000 都正常。

**Reply by changkaizhao** [reply]
搞个大内存的香港 VPS， 弄个GUI xrdp 远程桌面凑活用一下先 :(

**Comment by cross-hello** [comment]
תודה אתה
And תודה לאל

Feb 28, 2023 17:58:46 Johmy Engrish ***@***.***>:

**Comment by MakeFortuneQuietly** [comment]
看样子都是没套cdn导致封ip？如果墙能识别tls加密的流量进而封端口，那么挂不挂cdn在墙面前不都一样么。为什么套用了cdn就没事？

**Reply by wangdaye078** [reply]
因为一台CDN后面不是你的一台VPS，搞不好后面是一堆微软、亚马逊等等其他公司，所以封CDN应该会慎重得多。

**Comment by 1265578519** [comment]
https反而特征更明显不安全，而且https后一般是墙ip，普通的非加密流量被发现只是墙端口

**Comment by hamlinzheng** [comment]
IP可以Ping通，但是Telnet所有端口都没有回应，不知道开完会能不能恢复

**Comment by Uhtred009** [comment]
三个月 以后

hamlin ***@***.***> 于2023年3月6日周一 15:21写道：

**Comment by 1265578519** [comment]
没想到现在每天早上还是被ban端口，，下次试试纯tcp流量吧

**Reply by 1265578519** [reply]
测试了，纯tcp也不行，基本也是秒封，而且是和tls一样，直接封ip地址。

**Comment by Synthfrax** [comment]
今天我的也挂了，简单的tcp+tls

**Reply by 1265578519** [reply]
看来还是要ws了，因为只会封端口，换个端口就恢复了。

**Comment by 1265578519** [comment]
https://web.archive.org/web/20230331092744/https://www.aodun.com.cn/security_info/vpn
应该是这个原因了，怪不得大家都被秒墙，估计当地运营商防火墙已经部署上傲盾了，精准识别v2

**Reply by MakeFortuneQuietly** [reply]
所以一键脚本团灭，因为脚本没法设置cdn，不能抱跨国企业高级ip的大腿？

**Reply by HangJau** [reply]
> https://web.archive.org/web/20230331092744/https://www.aodun.com.cn/security_info/vpn 应该是这个原因了，怪不得大家都被秒墙，估计当地运营商防火墙已经部署上傲盾了，精准识别v2

是不是1号上的呀， 1号就不好使了。。。现在CDN苟延残喘

**Comment by 2efPer** [comment]
确实应该升级了，去年很稳，这几个月老换端口

**Comment by FreeSlaver** [comment]
真的没撒必要折腾，都是一路折腾过来的人，从ss到v2ray到trojan，
从nginx反代，套域名，套CDN，套cloudflare，套各种协议，都没撒用。
还是用别人的机场把，便宜稳定，撒事都不用操心，<a href="https://user.by.ltd/aff.php?aff=17672">bywave</a>。
你那折腾的几个小时玩会游戏，刷会视频，学习下，它不香吗？
这个机场我18年末用到现在，当初有个套餐是80块钱250gb流量，包终生。
后面遇到3次还是几次故障，只要时间够久都会赔偿，有一次好像是赔偿了100多g。
我就白嫖到了现在，还有70g流量。当时冲了100块，现在还有35块余额了。
专业的事情还是交给专业的人干把。

**Reply by rayleyboon** [reply]
就是，少折腾吧，本来想自建，查了下，还是有问题，懒得搞了。要不是VPN没有自动分流，我还继续用，哈哈。不过用了几年还有这么多流量，不咋上外网吗
